### ISSUE TITLE
notified lines are emitted to stderr, not stdout

### ISSUE THREAD

**Issue by justinsteven** [projectdiscovery/notify#134]
### Notify version:

```plain
% notify -version

             __  _ ___
  ___  ___  / /_(_) _/_ __
 / _ \/ _ \/ __/ / _/ // /
/_//_/\___/\__/_/_/ \_, / v1.0.1
                   /___/

                projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
[INF] Current Version: 1.0.1
```

### Current Behavior:

notify is emitting notified lines to stderr

Demo:

```plain
% notify -i /etc/issue >/dev/null

             __  _ ___
  ___  ___  / /_(_) _/_ __
 / _ \/ _ \/ __/ / _/ // /
/_//_/\___/\__/_/_/ \_, / v1.0.1
                   /___/

                projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
Using default provider config: /home/justin/.config/notify/provider-config.yaml
Debian GNU/Linux 11 \n \l
```

I expect to see "Debian GNU/Linux 11 \n \l" thrown away because stdout is redirected to `/dev/null`. Instead it's being printed, because it's on stderr.

```plain
% outfile=$(mktemp)

% notify -i /etc/issue 2>$outfile

% cat $outfile

             __  _ ___
  ___  ___  / /_(_) _/_ __
 / _ \/ _ \/ __/ / _/ // /
/_//_/\___/\__/_/_/ \_, / v1.0.1
                   /___/

                projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
Using default provider config: /home/justin/.config/notify/provider-config.yaml
Debian GNU/Linux 11 \n \l
```

Furthermore, suppressing the banner using `-silent` also suppresses notified lines:

```plain
% notify -i /etc/issue -silent
```

### Steps To Reproduce:

I expect to be able to do:

```plain
% do_work | notify | do_more_work
```

I essentially expect notify to take stdin, send notifications, and emit the lines back to stdout for further processing, ala httpx, nuclei, dnsx etc.

I also expect to be able to do `-silent` to suppress the banner without suppressing the output

Perhaps I'm misunderstanding the intent of notify though, and there is some reason why emitting its notified lines to stderr instead of stdout is preferred.

**Comment by justinsteven** [comment]
The notified line printer is doing format expansion too

```plain
% echo '%s' | ./notify

             __  _ ___
  ___  ___  / /_(_) _/_ __
 / _ \/ _ \/ __/ / _/ // /
/_//_/\___/\__/_/_/ \_, / v1.0.1
                   /___/

                projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
Using default provider config: /home/justin/.config/notify/provider-config.yaml
%!s(MISSING)
```

**Comment by justinsteven** [comment]
I tried to fix this via #130 but it got backed out at the last minute

The format string bug is now fixed, but `-silent` is still dropping notified lines

As of `173f91472fd73c410969d063776ff1a7224fd6d1`:

```
% ./notify -i /etc/issue

             __  _ ___
  ___  ___  / /_(_) _/_ __
 / _ \/ _ \/ __/ / _/ // /
/_//_/\___/\__/_/_/ \_, / v1.0.2-dev
                   /___/

                projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
Using default provider config: /home/justin/.config/notify/provider-config.yaml
Debian GNU/Linux 10 \n \l

% ./notify -i /etc/issue -silent
```

(No output is given for `-silent`)
